### ISSUE TITLE
Same results with IPAdapter in the chain and without?

### ISSUE THREAD

**Issue by purzbeats** [Shakker-Labs/ComfyUI-IPAdapter-Flux#5]
![image](https://github.com/user-attachments/assets/dc49505c-2233-44db-9203-d24bb4a12bc0)

Same results with IPAdapter on and off..?

**Comment by purzbeats** [comment]
Looks like the IPAdapter weight is maybe not being unloaded for the next run without it?

**Comment by purzbeats** [comment]
Yeah it only disables the IPAdapter when it's muted, but I think it should probably only load if it's in the model chain instead of sticking around.

**Comment by Lalimec** [comment]
Also the weight value doesnt seem to do anything.
![image](https://github.com/user-attachments/assets/29a6fb0e-20f8-427d-b5fb-33c8f04a7337)
![image](https://github.com/user-attachments/assets/a2e134d7-bbcd-42f0-a65e-4c2882aa60d2)

**Comment by purzbeats** [comment]
I think maybe the IPAdapter weight just needs to be unloaded when it's finished so that changes can happen on the chain.

It persists across multiple workflows/tabs as well, not even in the same workflow.

**Comment by philipy1219** [comment]
We will check it. Could you provide the current commit of ComfyUI you are using?
